keanu-python/matplotlibrc

```diff
@@ -0,0 +1 @@
+backend: Agg
```

Comment:
This is a good idea. I had previously worked around the problem locally by putting this in `~/.matplotlib/matplotlibrc`

Reply:
I believe it still yields warnings:

> UserWarning: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.

I don't know if it's possible to do better, but the warnings seem innocuous.

Reply:
Yep I looked briefly at getting rid of the warnings but couldn't seem to. Will look again.

Reply:
Have added a filter for the warnings now.

Reply:
nice